app/views/app/ddah.html.erb

```diff
@@ -105,7 +105,7 @@
       <p>You have accepted the DDAH conditions of this TAship.</p>
     </div>
     <div class='admonishment'>
-      <p><b>Important:</b> TO-DO</p>
+      <p><b>Important:</b> If you have any concerns with the allocations of duties, please discuss with the course coordinator.</p>
```

Comment:
Looks good.  Small change:

"If you have any concerns with the allocations of duties, please contact the course instructor as soon as possible."